python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -854,6 +855,17 @@ class DbtCliResource(ConfigurableResource):
         description="The path to the dbt executable.",
     )
 
+    @classmethod
+    def from_project(
+        cls,
+        project: DbtProject,
```

Comment:
Could consider standardizing on `project` over `project_dir` in the top level arguments, while still maintaining backcompat on the `project_dir` argument/property.

Not a fan of introducing a new initialization method for `DbtCliResource` -- I want to use the constructor if at all possible